specification/network/resource-manager/Microsoft.Network/2017-06-01/virtualNetwork.json

```diff
@@ -825,6 +826,13 @@
           "$ref": "./routeTable.json#/definitions/RouteTable",
           "description": "The reference of the RouteTable resource."
         },
+        "serviceEndpoints": {
```

Comment:
@amarzavery  @salameer this is well documented and we are okay with the breaking change.
it is read-write property.